boards/espressif/esp32_devkitc_wroom/doc/index.rst

```diff
@@ -77,6 +77,8 @@ Current Zephyr's ESP32-DevKitC-WROOM board supports the following features:
 +------------+------------+-------------------------------------+
 | PCNT       | on-chip    | qdec                                |
 +------------+------------+-------------------------------------+
```

Comment:
Please, rebase with main as the `Supported Features` documentation style was updated..